src/github.js

```diff
@@ -36,15 +37,32 @@ async function fetch_config() {
   const context = get_context();
   const octokit = get_octokit();
   const config_path = get_config_path();
+  const useLocal = get_use_local();
+  let content = '';
 
-  const { data: response_body } = await octokit.repos.getContent({
-    owner: context.repo.owner,
-    repo: context.repo.repo,
-    path: config_path,
-    ref: context.ref,
-  });
+  if (!useLocal) {
+    const { data: response_body } = await octokit.repos.getContent({
+      owner: context.repo.owner,
+      repo: context.repo.repo,
+      path: config_path,
+      ref: context.ref,
+    });
+
+    content = Buffer.from(response_body.content, response_body.encoding).toString();
+  } else {
+    try {
+      content = fs.readFileSync(config_path).toString();
+
+      if (!content) {
+        throw new Error();
+      }
+    } catch (error) {
+      core.debug(`Error when reading local file: ${error}`);
+
+      throw new Error('Local file missing');
```

Comment:
Ideally speaking, I'd define a custom error, but since the vanilla JavasScript isn't good at it (that's because I made https://github.com/necojackarc/extensible-custom-error, by the way 😆), let's define a constant so we don't forget to update the reference in `src/index.js`.

I'm happy for you to make a new file... as it'll have just one constant, any name should be fine for now such as `src/constants.js`, `src/error_messages.js`, etc. as long as it's easy to understand.

Reply:
Fully agreed, I was just trying not to make too many changes 😆 Updated